src/periphery/MagicApeLens.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.9;
+
+import "interfaces/IApeCoinStaking.sol";
+
+contract MagicApeLens {
+    uint256 constant PRECISION = 1e18;
+    uint256 constant BPS_PRECISION = 1e4;
+    address public immutable APE_COIN_STAKING_CONTRACT = 0x5954aB967Bc958940b7EB73ee84797Dc8a2AFbb9;
```

Comment:
use constant then, immutable is more when assigning them from the constructor so it's read only.